### PR TITLE
Upgrading charts and variable types

### PR DESCRIPTION
**Pull request by ConorH994**

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -5,4 +5,18 @@
...
+  <suppress>
+    <notes><![CDATA[
+   file name: guava-28.0-jre.jar
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/com\.google\.guava/guava@.*$</packageUrl>
+    <cve>CVE-2020-8908</cve>
+  </suppress>
+  <suppress>
+    <notes><![CDATA[
+   file name: httpclient-4.5.12.jar
```
Comment (dharmendra8186) [inline config/owasp/suppressions.xml:17]: do we need to suppress?

File: src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/BaseTest.java [hunk 2]
```diff
@@ -1,71 +1,77 @@
...
-            .password(POSTGRES)
-            .url(pg.getJdbcUrl(POSTGRES, POSTGRES))
-            .driverClassName("org.postgresql.Driver")
-            .build();
+            final Properties props = new Properties();
+            // Instruct JDBC to accept JSON string for JSONB
+            props.setProperty("stringtype", "unspecified");
+            connection = DriverManager.getConnection(pg.getJdbcUrl("postgres", "postgres"), props);
+            DataSource datasource = new SingleConnectionDataSource(connection, true);
+            Flyway.configure().dataSource(datasource)
```
Comment (dharmendra8186) [inline src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/BaseTest.java:62]: do we need this configuration as we already defined in application.yaml?

**Review by dharmendra8186 — COMMENTED** [review]
I left few review comments.Pls check.

File: src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/BaseTest.java (hunk 2 above)
Comment (ConorH994) [reply src/integrationTest/java/uk/gov/hmcts/reform/roleassignment/BaseTest.java:62]: Each of the integration tests fail when this config is not included here, I messed around with it a bit and couldn't see why it doesn't work without it.

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (ConorH994) [reply config/owasp/suppressions.xml]: I am upgrading now and checking

**Pull request closed without merging**